test/integration/openshift_health_checker/setup_container.yml

```diff
@@ -0,0 +1,34 @@
+---
+# Required vars:
+# * scenario = unique name for the container to be started
+# * image = name of the image to start in the container
+# Optional vars:
+# * l_groups = host groups that the container should be added to
+# * l_host_vars = any variables that should be added to the host
+
+- name: Start specified container
+  gather_facts: no
+  hosts: localhost
+  connection: local
+  vars:
+    container_name: openshift_ansible_test_{{ scenario }}
+  tasks:
+    - name: start container
+      docker_container:
+        name: "{{ container_name }}"
+        image: "{{ image }}"
+        command: sleep 1800
```

Comment:
If one test takes more than half an hour I would think we're in trouble anyway. This way if something goes wrong before the container teardown, it still expires eventually. So if you're developing and running tests you don't end up with a bunch of idle containers that run forever. That was my thought, but it's not a strong preference if you feel otherwise.